### ISSUE TITLE
What to put in "key" in config?

### ISSUE THREAD

**Issue by leonbohmann** [pwnlabs/gitlab2trello#6]
How do I find "key" entry in config.js?

**Comment by f3l1x** [comment]
Hi @leonbohmann 

https://github.com/f00b4r/gitlab2trello/blob/f92f37ff311dbf66aa20bbeca357d8b2a88c54b2/config.js#L6-L9

**Comment by leonbohmann** [comment]
Yes I know where to put it but I mean, how do I find the key of the board?

**Comment by f3l1x** [comment]
Oh, I am not sure it's 5 years since I used it. You can list your boards from API and parse ids manually.
